test/api/search_controller_test.rb

```diff
@@ -204,6 +204,19 @@ class SearchControllerTest < ActionDispatch::IntegrationTest
     assert_equal 'testing', json['studies'].first['term_matches'].first
     assert_equal 'API Test Study', json['studies'].last['term_matches'].first
 
+    # test author keyword search
+    @author = FactoryBot.create(:author, study: @study, last_name: 'Doe', first_name: 'john')
```

Comment:
lets move this author creation into the 'before(:all)' block so that they get created at a predictable time